Assets/Mirror/Components/NetworkProximityChecker.cs

```diff
@@ -94,7 +110,9 @@ public override bool OnRebuildObservers(HashSet<NetworkConnection> observers, bo
                             // (if an object has a connectionToClient, it is a player)
                             if (identity != null && identity.connectionToClient != null)
                             {
-                                observers.Add(identity.connectionToClient);
+                                // making sure the local player doesn't add himself twice on the observer list
```

Comment:
This comment is inaccurate. There is no observer list, its a HashSet, and as such cannot contain a duplicate entry.